packages/block-editor/src/text-input/edit.js

```diff
@@ -54,7 +54,7 @@ const EditTextInput = ( props ) => {
 			style={ { ...useColorStyles( attributes ) } }
 		>
 			<Sidebar { ...props } />
-			<FormInputWrapper.Label className="crowdsignal-forms-text-input-block__label">
+			<FormInputWrapper.Label className="crowdsignal-forms-checkbox-input-block__label">
```

Comment:
It looks like you changed the styling in the text input editor to use the checkbox's style.  Was that intentional?

Reply:
Yikes! No reason, will switch it back.